diesel_codegen/src/database_url.rs

```diff
@@ -29,8 +29,8 @@ fn load_dotenv_file() -> Result<(), String> {
 
 #[cfg(not(feature = "dotenv"))]
 fn load_dotenv_file() -> Result<(), String> {
-    Err(String::from("The dotenv feature is required to use strings starting \
-        with `dotenv:`"))
+    Err(String::from("diesel_codegen must be compiled with the dotenv feature \
+        to use strings starting with `dotenv:`"))
```

Comment:
This is fine, but if we're already editing this, I'd probably replace the "use strings" with something like "correctly substitute strings starting with `dotenv:`"